docs/getting-started/using-gnoland.md

```diff
@@ -0,0 +1,543 @@
+❯ gnoland secrets get
+
+[Node P2P Info]
+
+Node ID:  g16942vugc4g98j6gqfm94tnd4qwwgk8ee4nuntd
+[Validator Key Info]
+
+Address:     g1ukvflgxqehyrzwm0ulh7a9sn6hm7505dc0mwrj
+Public Key:  gpub1pggj7ard9eg82cjtv4u52epjx56nzwgjyg9zquv55ka63cpwqx56j22fesrevem54dqdlfy2s008l4ev2cvhnnyk9tdf0t
+[Last Validator Sign State Info]
+
+Height:  0
+Round:   0
+Step:    0
+```
+
+#### Flags
+- `-data-dir gnoland-data/secrets` - the directory from which to read the secrets
+
+### Verifying Secrets
+
+You can verify the existence and integrity of all node secrets using the 
+`secrets verify` subcommand. The command can be run for any specific secret,
+or for all of them at the same time.
+
+```bash
+gnoland secrets verify [flags] [<key>]
+```
+
+In case a specific node secret is missing or corrupt, the command will let you know.
+For example, if the `node_key.json` file was missing from the secrets directory,
+checking for it would result in the following output:
+
+```bash
+❯ gnoland secrets verify NodeKey 
+unable to read node p2p key, unable to read data, open gnoland-data/secrets/node_key.json: no such file or directory
+```
+
+#### Flags
+- `-data-dir gnoland-data/secrets` - the directory from which to read the secrets
+
+## Node Configuration
+
+The `gnoland config` command suite helps manage the gno.land node configuration.
+
+### Initializing Configuration
+
+To initialize the gno.land node configuration, you can use the following command:
+
+```bash
+gnoland config init [flags]
+```
+
+This command will initialize a configuration directory and file in the default
+path. The node is highly configurable. Some of the main configuration options
+which are available are listed below:
+- Database which is used for storing node data
+- Fast synchronization
+- Local paths for node secrets
+- Consensus configuration
+- Mempool configuration
+- Networking (P2P) configuration
+- RPC configuration
+- Telemetry
+
+To view the fully detailed configuration options, take a look at the generated
+`config.toml` file.
+
+#### Flags
+- `-data-dir gnoland-data/config` - the directory where the configuration file will be saved in
+
+### Getting Node Configuration
+
+The `config get` subcommand allows you to read specific configuration values
+from the `config.toml` file.
+
+```bash
+gnoland config get [flags] <key>
+```
+
+For example, fetching the RPC listener address of the
+node can be done the following way, considering that the listener address key is
+`laddr`, found under the `rpc` category:
+
+```bash
+gnoland config get rpc.laddr
+```
+
+A full configuration category, such as `consensus`, can also be read via this command:
+
+```bash
+gnoland config get rpc
+```
+
+For the RPC category, the default output for this subcommand will be a struct of
+all configuration fields inside it, similar to the following:
+
+```bash
+{ tcp://127.0.0.1:26657 [*] [HEAD GET POST OPTIONS] [Origin Accept Content-Type X-Requested-With X-Server-Time]  900 false 900 10s 1000000 1048576  }
```

Comment:
This is a bit confusing. The example went from rpc, to consensus, and back to rpc again. Is there an option to have the output labeled by field? That would be more helpful for people imo. Otherwise maybe omit the output; I'm no sure it adds much.